invenio_app_ils/mappings/v6/documents/document-v1.0.0.json

```diff
@@ -7,23 +7,32 @@
         "$schema": {
           "type": "keyword"
         },
-        "title": {
+        "_created": {
```

Comment:
This is what we saw with @topless, it should be needed because it `_created` and `_updated` is outside the metadata already created by Invenio automatically, but we need to double check it!
![Screenshot 2019-04-02 at 11 19 37](https://user-images.githubusercontent.com/2089455/55391240-432c8a00-5539-11e9-9b27-ac065a21eafd.png)

Reply:
Forget what I said here... it is Invenio changing the response.
![Screenshot 2019-04-02 at 15 49 02](https://user-images.githubusercontent.com/2089455/55407771-e643ca80-555e-11e9-93af-e5232fd7403b.png)